download-windows-esd

```diff
@@ -1,65 +1,65 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
Let's not introduce Bash as a dependency here. I would like to move toward portability.

Reply:
The autocomplete is separate from the tool. It's just an additional convenience I added—the tool can run without it.

The autocomplete _is_ dependent on Bash (and it's also been nice that it works on zsh too).

The tool isn't and will continue to be that way.